app/controllers/hosts_controller.rb

```diff
@@ -93,6 +93,7 @@ def new
   def clone
     @clone_host = @host
     @host = @clone_host.clone
+    @host.build = true
```

Comment:
What is we clone a host that is unmanaged? Maybe this should copy the build from the host that is being cloned or only set if the host is managed?